administrator/components/com_associations/views/association/tmpl/edit.php

```diff
@@ -33,7 +26,35 @@ function triggerSave() {
 			$('#left-panel').toggle();
 			$('#right-panel').toggleClass('full-width');
 		});
+
+		$('.btn-success').attr('onclick','return triggerSave()');
+
+		$('#reference-association').load(function (){
+			$('#reference-association').each(function () {
+				$(this).contents().find('.chzn-single').css('background', 'transparent');
+				$(this).contents().find('.chzn-single').css('background-color', '#eee');
+				$(this).contents().find('.controls').css( 'pointer-events', 'none' );
+    			$(this).contents().find('input').attr('disabled', 'disabled');
```

Comment:
in this case. i don't know if readonly or disabled
Anyway the tinymce has the readonly too. something like this
```editorobj.setMode('readonly');``` or ```readonly: true,``` on init
Also check the "Permissions" tab, those are not blocked.

But thinking out loud, maybe the user sees an error in the reference language and want to edit it too... maybe it should be editable too if needed. In that case another save button is needed. what do you think?